docs/intro-react.md

```diff
@@ -143,18 +143,21 @@ export default function Cat() {
 }
 ```
 
-Any JavaScript will work between curly braces, including functions like `{fullName("Rum", Tum", "Tugger")}`:
+Any JavaScript expression will work between curly braces, including function calls like `{getFullName("Rum", Tum", "Tugger")}`:
```

Comment:
The convention for function that return something is `get*`

Reply:
I almost feel like that would make a great

> note.

Reply:
This is more of a JS convention — can't really teach every one of them here! The reason for it is that unless you look at the definition, `fullName` actually appears to be a name itself. `getFullName` is more clearly a function.